test/fixtures/env.js

```diff
@@ -4,5 +4,6 @@ module.exports = {
   BEEP: false,
   BOOP: [ 'some', 'thing', 'that', 'goes', 'wow' ],
   BLEEP: 'false',
-  PING: [ 'ping', 'true', 2, 100 ]
+  PING: [ 'ping', 'true', 2, 100 ],
+  PONG: 'some,thing,that,goes,wow'
```

Comment:
I don't understand why this must be  encapsulated in quotes. The env.js is the expected result, right? So for PONG you expect a string without the quotes was my thought.